repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/pipeline/Launcher.java

```diff
@@ -160,6 +164,17 @@ private JSAP defineArgs() throws JSAPException {
         opt2.setHelp("Specify the file containing a list of projects that the pipeline should deactivate serialization when processing builds from.");
         jsap.registerParameter(opt2);
 
+        opt2 = new FlaggedOption("repairTools");
+        opt2.setLongFlag("repairTools");
+        String repairTools = StringUtils.join(RepairToolsManager.getRepairToolsName(), ",");
+        opt2.setStringParser(EnumeratedStringParser.getParser(repairTools.replace(',',';'), true));
+        opt2.setList(true);
+        opt2.setListSeparator(',');
+        opt2.setHelp("Specify one or several repair tools to use among: "+repairTools);
+        opt2.setRequired(true);
```

Comment:
I still didn't look the entire changes, but for what I see here, we will always need to give at least one repair tool as arg. Is that correct?

Reply:
By default the list of all available repair tools is given. But yes.